proposals/interface-types/working-notes/scenarios/directory.md

```diff
@@ -0,0 +1,243 @@
+# Directory Listing
```

Comment:
This scenario doesn't have any written-up description of how iteration/sequences/arrays work in `working-notes`.  Probably good to add a big bold banner saying it's speculative, ahead of working notes and Explainer.md text?